pkg/didcomm/crypto/crypter.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+// Crypter is an Aries envelope encrypter to support
+// secure DIDComm exchange of envelopes between Aries agents
+type Crypter interface {
+	// Encrypt a payload in an Aries compliant format
+	// returns:
+	// 		[]byte containing the encrypted envelope
+	//		error if encryption failed
+	Encrypt(payload string) ([]byte, error)
+	// Decrypt an envelope in an Aries compliant format
+	// returns:
+	// 		string containing the decrypted payload
+	//		error if encryption failed
+	// TODO add recipient's private key as an argument of Decrypt as it's needed for decrypting the message
+	Decrypt(envelope []byte) (string, error)
```

Comment:
I think the return value is better represented as []byte.

Reply:
will update

Reply:
I think I've already updated this to []byte